docs/processing_data.rst

```diff
@@ -9,6 +9,27 @@
 .. |filter_north| image:: images/filter_8.jpg
 .. |filter_select| image:: images/filter_9.jpg
 .. |filter_floor| image:: images/filter_10.jpg
+.. |branch_1| image:: images/branch_1.jpg
+.. |branch_2| image:: images/branch_2.jpg
+.. |branch_3| image:: images/branch_3.jpg
+.. |branch_4| image:: images/branch_4.jpg
+.. |flow_addfilter| image:: images/flow_addfilter.jpg
+.. |flow_definefilter| image:: images/flow_definefilter.jpg
+.. |flow_added| image:: images/flow_filteradded.jpg
+.. |flow_filterconfig| image:: images/flow_filterconfig.jpg
+.. |flow_filterpipeline| image:: images/flow_filterpipeline.jpg
```

Comment:
Image _flow_filterpipeline_ is not used in this document. Is it used elsewhere? The jpg appears to be a duplicate of _flow_filterdone_ except for the curve in one arrow.